net-misc/sslh/sslh-1.18-r1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+MY_P="${PN}-v${PV}"
+inherit toolchain-funcs systemd
+
+DESCRIPTION="Port multiplexer - accept both HTTPS and SSH connections on the same port"
+HOMEPAGE="http://www.rutschle.net/tech/sslh.shtml"
+SRC_URI="http://www.rutschle.net/tech/${PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~m68k ~mips ~s390 ~sh ~x86"
+IUSE="caps tcpd systemd"
+
+RDEPEND="caps? ( sys-libs/libcap )
+	tcpd? ( sys-apps/tcp-wrappers )
+	systemd? ( sys-apps/systemd )
```

Comment:
If sslh links to libsystemd.so, we could consider adding a subslot operator: ``systemd? ( sys-apps/systemd:= )``.

According to systemd's metadata.xml, the subslot will be incremented if there's an ABI break, so the operator will trigger a rebuild of sslh.